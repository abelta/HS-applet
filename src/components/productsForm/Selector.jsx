import React from 'react';
import PropTypes from 'prop-types';

const Selector = ({ children, onClick }) => <button onClick={onClick}>{children}</button>;

Selector.propTypes = {
  children: PropTypes.node.isRequired,
  onClick: PropTypes.func.isRequired,
};

export default Selector;
