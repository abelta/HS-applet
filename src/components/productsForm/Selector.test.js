import React from 'react';
import renderer from 'react-test-renderer';
import Selector from './Selector';

describe('Selector', () => {
  const tree = renderer.create(<Selector onClick={jest.fn}>Segment</Selector>).toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
