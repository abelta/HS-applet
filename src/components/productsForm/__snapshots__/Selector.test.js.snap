// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selector renders correctly 1`] = `
<button
  className="btn-flat waves-effect waves-light white-text blue"
  onClick={[Function]}
  style={
    Object {
      "borderRadius": 0,
    }
  }
>
  Segment
</button>
`;
